noir/noir-repo/acvm-repo/acvm_js/src/public_witness.rs

```diff
@@ -63,9 +63,9 @@ pub fn get_public_parameters_witness(
     let program: Program =
         Program::deserialize_program(&program).expect("Failed to deserialize circuit");
     let circuit = match program.functions.len() {
-        0 => return Ok(initial_witness),
+        0 => return Ok(JsWitnessMap::from(WitnessMap::new())),
```

Comment:
Why should we not return the initial witness here?

Reply:
I guess we still can, I was thinking if it is an empty circuit the witness is not necessarily useful

Reply:
These methods like `get_public_parameters_witness` and `get_return_witness` also require circuit information for the accurate witness.

Reply:
It made more sense to me mentally that if there is no circuit there is no witness

Reply:
I'm just thinking that a program which contains zero circuits is essentially the same as a program which contains an empty circuit.

Kinda moot I guess as we'll need to be updating this interface very soon however.

Reply:
Yeah this will change soon either way